website/docs/r/organization_module_sharing.html.markdown

```diff
@@ -10,8 +10,7 @@ description: |-
 
 Manage module sharing for an organization.
 
-~> **NOTE:** This resource requires using the provider with 
-an instance of Terraform Enterprise at least as recent as v202004-1.
+-> **NOTE:** `tfe_organization_module_sharing` is deprecated in favor of using `tfe_admin_organization_settings` which also allows the management of the global module sharing setting. They attempt to manage the same resource and are mutually exclusive.
```

Comment:
```suggestion
-> **NOTE:** `tfe_organization_module_sharing` is deprecated in favor of using `tfe_admin_organization_settings` which also allows the management of the global module sharing setting. Both arguments attempt to manage the same resource and are mutually exclusive.
```

Reply:
dropped the commit

Reply:
I'm confused how `tfe_organization_module_sharing` can be deprecated if `tfe_admin_organization_settings` is only available for TFE. What about the TFC side of things? Also, would it make sense to mention the tfe-only-ness in the docs too?

Reply:
Module sharing is not a feature of TFC, but I see the need to mention that it's TFE-only in the old `tfe_organization_module_sharing` docs. The new docs do mention that the `tfe_admin_organization_settings` is TFE-only.